src/components/atoms/image/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Image = ({ image }) => <img src={image} alt={'preview'} />
```

Comment:
we can improve this component with other props, like:
- **className**
- **sizes** & **srcset**: to support 1x, 2x and 3x images 
- **async**: to support asynchronous loading

Reply:
you can add a TODO into the code, but we need to implement it in the future.